kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/migration/MigrationJobAPIImpl.java

```diff
@@ -133,6 +137,32 @@ protected TableBasedPipelineJobInfo getJobInfo(final String jobId) {
         return new TableBasedPipelineJobInfo(jobMetaData, getJobConfiguration(jobConfigPOJO).getSourceTableName());
     }
     
+    @Override
+    public List<InventoryIncrementalJobItemInfo> getJobItemInfos(final String jobId) {
```

Comment:
Could keep it in `AbstractInventoryIncrementalJobAPIImpl` and extend it for common usage